packages/components/timeline/src/timeline.styles.ts

```diff
@@ -0,0 +1,181 @@
+import styled from '@emotion/styled';
+import {
+  TimelineGroupProps,
+  TimelineGroupStyledProps,
+  TimelineProps,
+  TimelineStyledProps,
+} from './timeline.types';
+
+export const TimelineContainerStyled = styled.div<
+  {
+    styled: Required<TimelineStyledProps>;
+  } & TimelineProps
+>`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border: 0.063rem solid ${({ styled }) => styled.borderColor};
+  background-color: ${({ styled }) => styled.backgroundColor};
+  border-radius: 0.25rem;
+  gap: 1rem;
+  transition: all 0.3s ease-in-out;
+  width: ${({ horizontally }) => (horizontally ? '25vw' : '40vw')};
+  height: 100%;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.tablet}px) {
+    height: 100% !important;
```

Comment:
We shouldn't set '!important' on properties. It will be difficult to style from outside.